jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/JibBuildRunner.java

```diff
@@ -278,8 +278,9 @@ public JibContainer runBuild()
       throw new BuildStepsExecutionException(message, ex);
 
     } catch (ExecutionException ex) {
-      String message = Verify.verifyNotNull(ex.getCause().getMessage()); // keep null-away happy
-      throw new BuildStepsExecutionException(message, ex.getCause());
+      String message = ex.getCause().getMessage();
```

Comment:
Can `getCause()` be null?

Reply:
In our settings, I believe no. `ExecutionException` is from futures. Actually, the Javadoc says

> Exception thrown when attempting to retrieve the result of a task that aborted by throwing an exception. This exception can be inspected using the `Throwable.getCause()` method.